### ISSUE TITLE
Circle CI Tests are failing on Master

### ISSUE THREAD

**Issue by shannonlal** [plotly/falcon-sql-client#333]
I submitted a pull requests and this was just an MD file (NEW_CONNECTION.MD) and when I did I noticed the Circle CI tests fail.  I looked at Circle CI and it looks like it has something to do the the DB2 database.  I have pasted the error message from Cirlcle CI here

```bash
(node:967) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: [IBM][CLI Driver] SQL30081N  A communication error has been detected. Communication protocol being used: "TCP/IP".  Communication API being used: "SOCKETS".  Location where the error was detected: "35.184.35.183".  Communication function detecting the error: "selectForConnectTimeout".  Protocol specific error code(s): "115", "*", "*".  SQLSTATE=08001

```

The failing test

```bash
 1) IBM DB2: connect succeeds:
     Error: Timeout of 90000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

**Comment by n-riesco** [comment]
Some of the sample databases are currently down. Sorry about that.
There is already someone looking into this issue.
We're also discussing the option of moving some of the sample databases into docker containers to be run locally.

**Comment by n-riesco** [comment]
Closed via #335
